src/ORM/TableRegistry.php

```diff
@@ -184,7 +184,7 @@ public static function get($alias, array $options = [])
         }
 
         if (isset(static::$_config[$alias])) {
-            $options += static::$_config[$alias];
+            $options = static::$_config[$alias] + $options;
```

Comment:
This will make existing config take precedence over options passed in argument, making passing options kinda useless as it will never override existing key.